env0/resource_azure_credentials_test.go

```diff
@@ -135,15 +135,18 @@ func TestUnitAzureCredentialsResource(t *testing.T) {
 	})
 
 	t.Run("validate missing arguments", func(t *testing.T) {
-		missingArgumentsTestCases := []resource.TestCase{
-			missingArgumentTestCase(resourceType, resourceName, map[string]interface{}{}, "client_id"),
-			missingArgumentTestCase(resourceType, resourceName, map[string]interface{}{}, "client_secret"),
-			missingArgumentTestCase(resourceType, resourceName, map[string]interface{}{}, "subscription_id"),
-			missingArgumentTestCase(resourceType, resourceName, map[string]interface{}{}, "tenant_id"),
-			missingArgumentTestCase(resourceType, resourceName, map[string]interface{}{}, "name"),
+		arguments := []string{
+			"client_id",
+			"client_secret",
+			"subscription_id",
+			"tenant_id",
+			"name",
 		}
-		for _, testCase := range missingArgumentsTestCases {
-			runUnitTest(t, testCase, func(mock *client.MockApiClientInterface) {
+
+		for _, argument := range arguments {
+			tc := missingArgumentTestCase(resourceType, resourceName, map[string]interface{}{}, argument)
```

Comment:
There are several places with similar fixes. Doing it in the old manner causes a parallel test inside a parallel, which is not allowed in Golang.